composer.lock

```diff
@@ -495,7 +495,7 @@
                 "MIT"
             ],
             "description": "eLife Sciences API client",
-            "time": "2017-10-23T09:04:22+00:00"
+            "time": "2017-10-23 09:04:22"
```

Comment:
Are you using an out-of-date version of Composer?

Reply:
yep. sorted